documentation/kubernetes.md

```diff
@@ -152,6 +166,75 @@ spec:
 
 </details>
 
+<details>
+  <summary>.NET Monitor 8+</summary>
+
+```yaml
+# Tell us about your experience using dotnet monitor: https://aka.ms/dotnet-monitor-survey
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deploy-exampleapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: exampleapp
+  template:
+    metadata:
+      labels:
+        app: exampleapp
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: app
+        image: mcr.microsoft.com/dotnet/samples:aspnetapp
+        imagePullPolicy: Always
+        env:
+        - name: ASPNETCORE_URLS
+          value: http://+:80
```

Comment:
Per @schmittjoseph's testing, cannot bind to port 80 as non-root

Reply:
8080 is used in the non-root .NET Docker images by default